### PR TITLE
Do not convert upload progress to integer

### PR DESCRIPTION
**Pull request by ngocdaothanh**
Currently, the progress is an integer number: 0%, 1%, 2%...

When I upload a big file (5GB etc.), the integer number doesn't change much, causing bad user experience.

This PR changes the progress to a float number. How the number is formatted is up to the users of the library, giving more flexibility.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -288,7 +288,7 @@ Uploader.prototype._upload = function() {
     xhr.upload.onprogress = function(event) {
 
       if (event.lengthComputable) {
-        var progress = (event.loaded / event.total * 100 | 0);
+        var progress = event.loaded / event.total * 100;
```
Comment (ngocdaothanh) [inline lib/index.js]: This is the only change in this file. Others are only removing trailing spaces.

File: README.md [hunk 2]
```diff
@@ -150,11 +150,13 @@ If upload progress support is available in `XMLHttpRequest`, then this event is
 uploader.on('upload:progress', function(progress) {
 
   /**
-   * `progress` is a Number between 0 and 100.
-   * 
+   * `progress` is a float Number between 0.0 and 100.0.
+   *
    * Here you can, for example, increment a progress bar.
+   *
+   * You can format the number using `Math.floor(progress)`, `progress.toFixed(2)` etc.
```
Comment (ngocdaothanh) [inline README.md]: Added instruction about formatting progress.

Comment (ngocdaothanh) [reply README.md]: This is the only change in this file. Others are only removing trailing spaces.

**Comment by ngocdaothanh** [comment]
Please kindly release a new version after merging. Thanks!

**Comment by ngocdaothanh** [comment]
@mpangrazzi Hi, any feedback?

**Comment by ngocdaothanh** [comment]
Thanks a lot!

Can you please release a new version?
The last release was 3 years ago:
https://www.npmjs.com/package/html5-uploader

**Comment by mpangrazzi** [comment]
I've just published `0.2.0`. Keep in mind that IMHO this package could be still valid to use, but for sure it requires a major refactoring :)

**Pull request merged**